Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -73,7 +73,7 @@ static const char* dumpReadyState(WebCore::MediaPlayer::ReadyState readyState)
 }
 #endif // GST_DISABLE_GST_DEBUG
 
-GST_DEBUG_CATEGORY(webkit_mse_debug);
+GST_DEBUG_CATEGORY_STATIC(webkit_mse_debug);
```

Comment:
Shouldn't the variable be named `webkit_mse_player_debug` for consistency?

Reply:
Yes, makes sense, thanks for the review :)